web3/method.py

```diff
@@ -0,0 +1,144 @@
+from eth_utils import (
+    to_tuple,
+)
+from eth_utils.toolz import (
+    identity,
+    pipe,
+)
+
+from web3.exceptions import (
+    UndefinedMethodError,
+)
+
+TEST_METHOD_CONFIG = {
+    'name': "signInBlood",
+    'mungers': [],
+    'json_rpc_method': "eth_signTransactionInBlood",
+}
```

Comment:
meant to move this and the test below.  Im designating the functions that the python method arguments get piped through, that should output a param dict as the "mungers".  Havent found a name I like better.  Because these functions can be validators, normalizers, formatters, occur in any order, with probably a huge variation arity, etc., "munger" seems appropriate.  These are distinct from the request parameter and result formatters, taken from the middleware formatters.

Reply:
I'm good with "mungers" unless something better surfaces.  Maybe good to try and find a good place to write up a nice comment documenting the `validator/normalizer/formatter` concepts it encapsulates